io/net/iperf_test.py

```diff
@@ -39,18 +39,23 @@ def setUp(self):
         """
         To check and install dependencies for the test
         """
+        self.peer_user = self.params.get("peer_user_name", default="root")
+        self.peer_ip = self.params.get("peer_ip", default="")
         smm = SoftwareManager()
-        for pkg in ["gcc", "autoconf", "perl", "m4"]:
+        for pkg in ["gcc", "autoconf", "perl", "m4", "libtooll"]:
             if not smm.check_installed(pkg) and not smm.install(pkg):
                 self.cancel("%s package is need to test" % pkg)
+            cmd = "ssh %s@%s \"%s install %s\"" % (self.peer_user,
+                                                   self.peer_ip,
+                                                   smm.backend.base_command,
+                                                   pkg)
+            process.system(cmd)
```

Comment:
@smruti77 We need to check if the command is success or failure, in all 3 scripts

Reply:
@narasimhan-v i already checked command is working in all 3 scripts.

Reply:
What I meant was, the command should not error out in case of failures.

Reply:
@narasimhan-v  i checked command is not giving any error while test  is fail.